coupling/delwaq/README.md

```diff
@@ -0,0 +1,37 @@
+# Ribasim Delwaq coupling
+
+
+## Steps
+Choose a testmodel and run it. For example, running the basic model with ribasim_cli:
+```bash
+./build/ribasim_cli/bin/ribasim generated_testmodels/basic/ribasim.toml
+```
+
+Run `gen_delwaq.py` to couple the models. Open a terminal in the directory of Ribasim
+```python
+python ./coupling/delwaq/gen_delwaq.py
+```
+The script can:
+- Open an existing Ribasim model (hardcoded to `generated_testmodels/basic/ribasim.toml` for now)
+- Convert the metadata, topology and substances to Delwaq format and put it into the `model` folder
+
+You now should manually run Delwaq. You can do so with the Docker image.
```

Comment:
Good to make it clear from the start that this only works for Deltares employees.